src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,105 @@
+    @NullAndEmptySource
+    void emptyOrNull(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isZero();
+    }
+
+    @DisplayName("blank 문자열을 입력하는 경우 0을 반환한다")
+    @ParameterizedTest
+    @ValueSource(strings = {" ", "  ", "   "})
+    void blank(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isZero();
+    }
+
+    @DisplayName("숫자 하나를 문자열로 입력할 경우 해당 숫자를 반환한다")
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "3"})
+    void oneNumber(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isEqualTo(Integer.parseInt(text));
+    }
+
+    @DisplayName("숫자 두개를 컴마(,) 구분자로 입력할 경우 두 숫자의 합을 반환한다")
+    @ParameterizedTest
+    @ValueSource(strings = {"1,2"})
+    void twoNumberWithComma(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isEqualTo(3);
+    }
+
+    @DisplayName("구분자를 컴마(,) 이외에 콜론(:)을 사용할 수 있다")
+    @ParameterizedTest
+    @ValueSource(strings = {"1,2:3"})
+    void colon(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isEqualTo(6);
+    }
+
+    @DisplayName("//와 \\n문자 사이에 커스텀 구분자를 지정할 수 있다")
+    @ParameterizedTest
+    @ValueSource(strings = {"//;\n1;2;3"})
+    void customDelimiter(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isEqualTo(6);
+    }
+
+    @DisplayName("커스텀 구분자와 기존 구분자를 함께 사용할 수 있다")
+    @ParameterizedTest
+    @ValueSource(strings = {"//;\n1,2:3;4"})
+    void withCustomDelimiter(final String text) {
+        final int expect = calculator.calculate(text);
+        assertThat(expect).isEqualTo(10);
+    }
+
+    @DisplayName("음수를 전달하는 경우 RuntimeException 예외 처리를 한다")
+    @ParameterizedTest
+    @ValueSource(strings = {"2:3:-1"})
+    void negativeNumber(final String text) {
+        assertThatExceptionOfType(RuntimeException.class)
+            .isThrownBy(() -> calculator.calculate(text));
+    }
+
+    @DisplayName("숫자 이외의 값이 전달 되는 경우 RuntimeException 예외 처리를 한다")
+    @ParameterizedTest
+    @ValueSource(strings = {"2:3:a"})
+    void notNumber(final String text) {
+        assertThatExceptionOfType(RuntimeException.class)
+            .isThrownBy(() -> calculator.calculate(text));
+    }
+
+    @Test
+    void whiteSpace() {
+        assertAll(
+            () -> {
+                assertThat(StringUtils.isEmpty(null)).isTrue();
+                assertThat(StringUtils.isEmpty("")).isTrue();
+                assertThat(StringUtils.isEmpty(" ")).isFalse();
+
+                assertThat(org.springframework.util.StringUtils.hasText(null)).isFalse();
+                assertThat(org.springframework.util.StringUtils.hasText("")).isFalse();
+                assertThat(org.springframework.util.StringUtils.hasText(" ")).isFalse();
+            }
+        );
```

Comment:
```suggestion
        assertAll(
            () -> assertThat(StringUtils.isEmpty(null)).isTrue(),
            () -> assertThat(StringUtils.isEmpty("")).isTrue(),
            ...
        );
```
위처럼 정의해야 검증할 테스트가 실패하더라도 중지하지 않고 남아있는 assert구문이 실행될 수 있을것 같아요 😄